app/services/required_document_category_analyser.rb

```diff
@@ -44,4 +51,8 @@ def has_opponents_application?
   def has_parental_responsibility?
     @application.proceedings.any? { |proceeding| proceeding.relationship_to_child.in?(%w[court_order parental_responsibility_agreement]) }
   end
+
+  def local_authority_assessed?
+    @application.proceedings.any? { |proceeding| proceeding.child_care_assessment&.assessed? }
+  end
```

Comment:
nit: naming wise, and for consistency with similar methods in here, I prefer has_local_authority_assessment?